src/test/ui/consts/const-eval/ub-int-array.stderr

```diff
@@ -0,0 +1,45 @@
+error[E0080]: it is undefined behavior to use this value
+  --> $DIR/ub-int-array.rs:15:1
+   |
+LL | / const UNINIT_INT_0: [u32; 3] = unsafe {
+LL | |
+LL | |
+LL | |     [
+...  |
+LL | |     ]
+LL | | };
+   | |__^ type validation failed: encountered uninitialized bytes at [0]
```

Comment:
Maybe we should dump the memory of constants at some point and have diagnostics point into them (but in general it's useless for validation to point to the entire constant's span)

Reply:
Yeah it's not great -- but also pre-existing.